NanoProd/crab/Sample_exceptions.yaml

```diff
@@ -0,0 +1,577 @@
+ST_t-channel_top_4f_InclusiveDecays:
+  - Run3_2023
+  - Run3_2023BPix
+ST_s-channel_4f_leptonDecays:
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+ST_s-channel_4f_hadronicDecays:
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+^ST_tW_(anti|)top_(2L2Nu|4Q|LNu2Q)(_ext1|): 
+  - Run2_2016
+  - Run2_2016_HIPM
+  - Run2_2017
+  - Run2_2018
+^ST_tW_(anti|)top_(2L2Nu|4Q|LNu2Q)_ext1: 
+  - Run3_2023
+  - Run3_2023BPix
+ST_tW_antitop_LNu2Q:
+  - Run3_2023
+^ST_s-channel_(anti|)top_4f_leptonDecays: 
+  - Run2_2016
+  - Run2_2016_HIPM
+  - Run2_2017
+  - Run2_2018
+  - Run3_2023
+  - Run3_2023BPix
+
+# Triboson
+^(WWW_4F|WWZ_4F|WZZ|ZZZ)_ext1:
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+
+# Z
+^ZJetsToNuNu_HT-(400To800|800To1500|1500To2500):
+  - Run2_2016
+  - Run2_2016_HIPM
+  - Run2_2017
+  - Run2_2018
+^ZJetsToNuNu_HT-(400To600|600To800|800To1200|1200To2500):
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+^ZJetsToQQ_HT-(200to400|400to600|600to800|800toInf): 
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+^EWK_(ZTo2L|ZTo2Nu): 
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+
+# W
+WtoLNu_amcatnloFXFX:
+  - Run3_2023
+  - Run3_2023BPix
+WtoLNu_madgraphMLM_ext1:
+  - Run2_2016
+  - Run2_2016_HIPM
+  - Run2_2017
+  - Run2_2018
+  - Run3_2023
+  - Run3_2023BPix
+^(WtoLNu_0J|WtoLNu_1J|WtoLNu_2J)-amcatnloFXFX:
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+^WtoLNu_HT-(100To400|400To800):
+  - Run2_2016
+  - Run2_2016_HIPM
+  - Run2_2017
+  - Run2_2018
+^WtoLNu_HT-(70To100|100To200|200To400|400To600|600To800|800To1200|1200To2500|2500ToInf):
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+^WtoLNu_HT-(1200To2500|2500ToInf|400To600|600To800|800To1200)_ext1:
+  - Run2_2016
+  - Run2_2017
+  - Run2_2018
+WtoLNu_HT-70To100_ext1:
+  - Run2_2016
+  - Run2_2016_HIPM
+  - Run2_2018
+WGToLNuG:
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+^EWK_(WplusToLNu|WminusToLNu):
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+
+# DY
+DYJetsToLL_M-50: 
+  - Run2_2016
+  - Run2_2016_HIPM
+  - Run2_2017
+  - Run2_2018
+  - Run3_2022
+  - Run3_2023
+  - Run3_2023BPix
+DYJetsToLL_M-10to50-madgraphMLM:
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+DYto2L_M-50_madgraphMLM_ext1:
+  - Run2_2016
+  - Run2_2016_HIPM
+  - Run3_2023
+  - Run3_2023BPix
+DYTo2L_MLL-4to50: 
+  - Run2_2016
+  - Run2_2016_HIPM
+  - Run2_2017
+  - Run2_2018
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+DYto2TautoMuTauh_M-50:
+  - Run2_2016
+  - Run2_2016_HIPM
+  - Run2_2017
+  - Run2_2018
+^DYJetsToLL_LHEFilterPtZ: 
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+^DYJetsToTauTauToMuTauh-madgraphMLM:
+  - Run3_2022
+  - Run3_2022EE
+  - Run3_2023
+  - Run3_2023BPix
+^DYJetsToLL_M-(100to200|200to500|500to700|700to800|800to1000|1000to1500|1500to2000|2000to3000)-amcatnloFXFX: 
```

Comment:
True, but there are `DYto2E_MLL-*`, `/DYto2Mu_MLL-*` and `/DYto2Tau_MLL-*` powheg samples that we can use.